pkmodel/tests/test_protocol.py

```diff
@@ -10,5 +10,71 @@ def test_create(self):
         """
         Tests Protocol creation.
         """
-        model = pk.Protocol()
-        self.assertEqual(model.time, 1)
+        with self.assertRaises(TypeError):
+            protocol = pk.Protocol('string')
+        protocol = pk.Protocol()
+        self.assertEqual(protocol.name, 'Protocol-initial_dose=1.0-time_span=1.0')  # noqa: E501
+
+    def test_encapsulation(self):
+        """
+        Tests Protocol values are encapsulated.
+        """
+        protocol = pk.Protocol()
+        with self.assertRaises(AttributeError):
+            protocol.Name
+        with self.assertRaises(AttributeError):
+            protocol.Initial_dose
+        with self.assertRaises(AttributeError):
+            protocol.Time_span
+        with self.assertRaises(AttributeError):
+            protocol._Name
+        with self.assertRaises(AttributeError):
+            protocol._Initial_dose
+        with self.assertRaises(AttributeError):
+            protocol._Time_span
+        with self.assertRaises(AttributeError):
```

Comment:
Don't lines 35-40 just repeat lines 29-34?

Reply:
One sequence is trying to access private variables starting with a single _.
The other is trying to access private variables starting with a double __.

Reply:
Ah makes sense, no issues then. I'll approve the merge.